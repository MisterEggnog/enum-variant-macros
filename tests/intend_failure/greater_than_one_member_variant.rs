use enum_variant_macros::*;
#[derive(FromVariants)]
enum Variants {
	Int(i32),
	Float(f32),
	Point(i32, i32),
}
fn main() {
}
