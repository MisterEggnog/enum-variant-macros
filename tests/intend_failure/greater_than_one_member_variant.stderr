error: FromVariants failed for Variants::Point, does not currently support variants that are larger than 1
 --> tests/intend_failure/greater_than_one_member_variant.rs:6:8
  |
6 |     Point(i32, i32),
  |           ^^^
